src/plugins/dashboard/public/dashboard_app/_dashboard_app_strings.ts

```diff
@@ -26,12 +26,26 @@ export const dashboardReadonlyBadge = {
 };
 
 export const dashboardManagedBadge = {
-  getTooltip: () =>
+  getText: () =>
     i18n.translate('dashboard.badge.managed.tooltip', {
-      defaultMessage: 'Elastic manages this dashboard. Clone it to make changes.',
+      defaultMessage: 'Elastic manages this dashboard. Duplicate it to make changes.',
     }),
 };
 
+export const managedText = i18n.translate('dashboard.managedContentBadge.text', {
+  defaultMessage: 'Managed',
+});
+
+export const text = dashboardManagedBadge.getText();
+
+export const buttonText = i18n.translate('dashboard.managedContentPopoverFooterText', {
+  defaultMessage: 'Duplicate this dashboard',
+});
+
+export const cancelButtonText = i18n.translate('dashboard.managedContentPopoverFooterCancelText', {
+  defaultMessage: 'Cancel',
+});
```

Comment:
Can you explain why these are exported separately rather than just including them under `dashboardManagedBadge`? 🙇